Frontend/src/components/salon-components/salonEmployees.tsx

```
@@ -11,19 +11,26 @@ import { useEmployeeTechnicianActions, useEmployeeTechnicianState } from "@/prov
 import "@ant-design/v5-patch-for-react-19";
 import { IEmployeeTechnician } from "@/providers/employeeTechnician-provider/context";
 
-
-
-const SalonEmployees = ({ employeeTechnicians: passedEmployees }: { employeeTechnicians?: IEmployeeTechnician[] }) => {
+const SalonEmployees = ({}: { employeeTechnicians?: IEmployeeTechnician[] }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)